Gems/OpenParticleSystem/Tools/OpenParticleSystemEditor/Window/ParticleGraphicsView.h

```diff
@@ -8,15 +8,15 @@
 
 #pragma once
 #if !defined(Q_MOC_RUN)
-#include <QGraphicsView>
```

Comment:
As a clean code practice I prefer separating QT includes from other includes, as you sometimes have to push/pop compiler warnings for them

Reply:
I moved most out of the #if, one was causing some build issues.